jsonParser/Organo.py

```diff
@@ -61,7 +104,24 @@ def __getType(self, text, type) -> object:
                     continue
         return lista
 
-    def __createJsonInfoList(self, infoList, voti_totali, listOfSeats) -> object:
+    def __supportExtract(self, mainList, listOfSeats, listOfNot, secondaryList) -> None:
+        listSupport = []
+        k = 0
+        for v in mainList:
+            if len(listOfNot) > 0: 
+                if listOfNot[0]-1 != k:
+                    secondaryList["seggio_n_" + str(listOfSeats[k])] = int(v)
+                    k += 1
+                else:
+                    listSupport.append(listOfNot.pop(0))
+            else:
+                secondaryList["seggio_n_" + str(listOfSeats[k])] = int(v)
+                k += 1
```

Comment:
```suggestion
            if len(listOfNot) > 0 and listOfNot[0]-1 == k: 
                    listSupport.append(listOfNot.pop(0))
            else:
                secondaryList["seggio_n_" + str(listOfSeats[k])] = int(v)
                k += 1
```

Praticamente ciò che c'è nell'else in basso lo ripeti esattamente identico dentro il secondo if, tanto vale che fai una sola condizione per `listSupport.append(listOfNot.pop(0))`